library/init/meta/interaction_monad.lean

```diff
@@ -94,6 +94,14 @@ meta def interaction_monad.orelse' {α : Type u} (t₁ t₂ : m α) (use_first_e
 meta instance interaction_monad.monad_fail : monad_fail m :=
 { fail := λ α s, interaction_monad.fail (to_fmt s), ..interaction_monad.monad }
 
+@[inline]
+meta def interaction_monad.bracket {α β γ} (x : m α) (inside : m β) (y : m γ) : m β :=
```

Comment:
```suggestion
meta def interaction_monad.finally {α β γ} (x : m α) (inside : m β) (y : m γ) : m β :=
```

Reply:
I chose this name to be conform to this naming scheme: https://hackage.haskell.org/package/base-4.12.0.0/docs/Control-Exception.html#v:bracket_. `finally` guarantees that an action will be run at the end of the computation but `bracket` gives us a preamble as well.

Reply:
Ok I see, so `bracket x y z = (x *> finally y z)`.  I didn't see the `x` at first.  Got it now.